eng/testing/performance/android_scenarios.proj

```diff
@@ -39,6 +39,12 @@
       <Command>$(Python) test.py devicestartup --device-type android --package-path pub\HelloAndroid.apk --package-name net.dot.HelloAndroid --scenario-name &quot;%(Identity)&quot; $(ScenarioArgs)</Command>
       <PostCommands>$(Python) post.py</PostCommands>
     </HelixWorkItem>
+    <HelixWorkItem Include="Build Time - Android HelloWorld">
+      <PayloadDirectory>$(WorkItemDirectory)</PayloadDirectory>
+      <PreCommands>cd $(ScenarioDirectory)helloandroid;mkdir traces;copy %HELIX_WORKITEM_ROOT%\androidHelloWorldBinlog\msbuild.binlog traces</PreCommands>
+      <Command>$(Python) test.py devicebuild --scenario-name &quot;%(Identity)&quot; $(ScenarioArgs) --binlog-path msbuild.binlog</Command>
```

Comment:
I would maybe choose a different name, e.g., `buildtime` instead `devicebuild` as it is not related to the physical mobile device.